.env

```diff
@@ -0,0 +1,4 @@
+KEYCLOAK_REALM=nordmart
+KEYCLOAK_URL=http://192.168.99.100:8180/auth/
+KEYCLOAK_CLIENT_BACKEND=stakater-nordmart-backend
+KEYCLOAK_CLIENT_SECRET_KEY=7672aea2-81ff-4aa2-affe-daf3bad10334
```

Comment:
Use `application.properties` ? No hard-coding of URLs

Reply:
Done.